Makefile

```diff
@@ -43,7 +45,6 @@ include vendor/github.com/gardener/gardener/hack/tools.mk
 .PHONY: start
 start:
 	@LEADER_ELECTION_NAMESPACE=garden GO111MODULE=on go run \
```

Comment:
Do we still need `GO111MODULE=on`?

@MartinWeindel removed those entries in similar pull requests for other extensions, e.g. https://github.com/gardener/gardener-extension-shoot-networking-filter/pull/111.